Cargo.toml

```diff
@@ -37,7 +37,14 @@ std = [
     'consensus-authorities/std',
     'grandpa/std',
     'finality-tracker/std',
+    'runtime-example-module/std',
 ]
+
+[dependencies.runtime-example-module]
+git = 'https://github.com/joystream/runtime-example-module.git'
+default_features = false
+rev = 'a14d1433ab9691170cd0b68fd6a82197ee7c61b9'
+
```

Comment:
During development before pushing a repo you might also want to use a path dependency:

```
[dependencies.runtime-example-module]
path = '../path-to-local-repo/example-module'
default_features = false
```

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies